src/bolos/cx_ec_domain.c

```diff
@@ -23,6 +23,8 @@ static uint8_t const C_cofactor_1[] = { 0, 0, 0, 1 };
 static uint8_t const C_cofactor_4[] = { 0, 0, 0, 4 };
 static uint8_t const C_cofactor_8[] = { 0, 0, 0, 8 };
 
+static int const NID_GENERIC = 0;
```

Comment:
I think `NID_undef` is already defined to `0` and be used instead of `NID_GENERIC`.

Reply:
ACL